backend/infrahub/workflows/utils.py

```diff
@@ -26,7 +27,7 @@ async def add_tags(
     namespace: bool = True,
     db_change: bool = False,
 ) -> None:
-    client = get_client(sync_client=False)
+    client = get_client(httpx_settings={"verify": get_http().verify_tls()}, sync_client=False)
```

Comment:
It's a bit confusing in the codebase that we use `get_client()`  where we sometimes want an InfrahubClient and in other locations we want a Prefect client. Did you just update this call as a hotspot where you needed this the most? Wondering if we should have a helper function to get the prefect client where we can update these settings in one location. I think we're good to go for this PR but that we should consider the approach for this for all the other calls.

Reply:
> Did you just update this call as a hotspot where you needed this the most?

Exactly, `add_tags()` is called within almost all workflows.
I agree it's confusing to have `get_client` for both InfrahubClient and PrefectClient...